.gitignore

```diff
@@ -337,4 +337,6 @@ coverage.cobertura.xml
 
 .docs/
 
-local.settings.json
\ No newline at end of file
+local.settings.json
+
+*.feature.cs
```

Comment:
It is no longer necessary. (I think it has been unnecessary for quite a long time actually. Ever since they moved the generation of these things out of the SpecFlow VS extension and into the build system.) In the long run I want to get rid of them across all projects.